webapp/src/pages/ChatRoom/MessageInput.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useRef, useState } from 'react';
```

Comment:
Don't you think maybe this MessageInput could be a component?

Reply:
What do we think about creating a components directory inside the `ChatRoom` directory? Because these two components are specific to this page. Something like:

```
ChatRoom.tsx
ChatRoom.css
components/
    index.ts
    Messages/
        Messages.tsx
        Messages.css
    MessageInput/
        MessageInput.tsx
        MessageInput.css
```

Reply:
The message input seems fairly useful to me in different pages, not just here. But we don't have an use case just yet, so maybe it's okay.

Besides that, I don't have a strong opinion on the topic. Maybe the others have opinions.